src/js/ImagePreview.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import css from '../stylesheets/button.css';
+
+export default class ImagePreview extends React.Component {
+
+   /**
+      * Find out if image src is available
+      * If not hide the image area otherwise show the image
+      * @return {String}
+   */
+   toggleImage () {
+      let image = this.props.imgSrc,
```

Comment:
Let's change this around and instead of passing a string data url to this component you pass File Object and convert it to a data url string in this component. It'll make it more modular.